db/migrate/20170124043154_create_customers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCustomers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :customers do |t|
+      t.string :first_name
+      t.string :last_name
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
One thing I forgot about is to use Josh's recommendation to use text instead of string. Thoughts on making everything a text data type going forward? @ski-climb